src/grt/src/fastroute/src/RipUp.cpp

```diff
@@ -326,6 +326,71 @@ bool FastRouteCore::newRipupCheck(const TreeEdge* treeedge,
   }
 }
 
+bool FastRouteCore::newRipupCheck(const TreeEdge* treeedge,
```

Comment:
This looks to be a copy of the existing newRipupCheck.  Is it possible to avoid duplicating the code?

Reply:
Yes it can be one function, I created a new one because this fuction is used in other places than mazeRouteMSMD, and it should the added case should't be analized outside it, and at the time there was no way to difference them, but now there is, and I can change it.